jmx/src/main/java/uk/gov/justice/services/jmx/MBeanInstantiator.java

```diff
@@ -0,0 +1,49 @@
+package uk.gov.justice.services.jmx;
+
+
+import static java.lang.management.ManagementFactory.getPlatformMBeanServer;
+
+import javax.annotation.PostConstruct;
+import javax.annotation.PreDestroy;
+import javax.ejb.Singleton;
+import javax.ejb.Startup;
+import javax.inject.Inject;
+import javax.management.InstanceAlreadyExistsException;
+import javax.management.InstanceNotFoundException;
+import javax.management.MBeanRegistrationException;
+import javax.management.MBeanServer;
+import javax.management.NotCompliantMBeanException;
+
+@Singleton
+@Startup
+public class MBeanInstantiator {
+
+    @Inject
+     MBeanRegistry mBeanRegistry;
```

Comment:
private

Reply:
We had a team decision, sometime ago, to make all injected classes package protected.  I believe we should continue to do so for consistency.